### ISSUE TITLE
Error: Cannot resolve module 'react/lib/CSSPropertyOperations'

### ISSUE THREAD

**Issue by Kikobeats** [algolia/instantsearch.js#1577]
```
ERROR in ./~/react-instantsearch/src/core/styling.js
Module not found: Error: Cannot resolve module 'react/lib/CSSPropertyOperations' in /Users/josefranciscoverdugambin/Projects/windtoday/windtoday-app/node_modules/react-instantsearch/src/core
 @ ./~/react-instantsearch/src/core/styling.js 19:29-71
```

React version: `15.4.0`

It looks that the core will not offer this method any more.

Related: https://github.com/algolia/instantsearch.js/blob/v2/packages/react-instantsearch/src/core/styling.js#L12

**Comment by vvo** [comment]
completely missed that line, do not even know what it does.

**Comment by vvo** [comment]
We are in the process of completely removing a lot of the styling infrastructure and just default to CSS class names. In the meantime, can you use React 0.13? We will need to fix that asap because react 0.14 is out
